content/en/agent/cluster_agent/external_metrics.md

```diff
@@ -119,6 +119,140 @@ Every 30 seconds, Kubernetes queries the Datadog Cluster Agent to get the value
 
 **Note**: Running multiple Cluster Agents will raise API usage. The Datadog Cluster Agent completes 120 calls per hour for approximately 45 HPA objects in Kubernetes. Running more than 45 HPAs will increase the number of calls when fetching metrics from within the same org.
 
+## Autoscaling with custom queries using DatadogMetric CRD (Cluster Agent >= v1.7.0)
+
+In some autoscaling scenarios, you may want to provide your own Datadog query to scale on, that's possible thanks to the `DatadogMetric` CRD and Datadog Cluster Agent >= 1.7.0.
+
+### Custom query requirements
+
+In order for autoscaling to work properly, the query must follow some rules:
+* The query **must** be syntactically correct, otherwise it will prevent the update of **ALL** metrics used for autoscaling.
+* The query result **must** yield only 1 serie in output (otherwise, the results are considered invalid).
+* The query **should** yield at least two timestamped points (it's possible to use a query that returns a single point, though in this case, autoscaling may use incomplete points).
+
+**Note**: While the query is arbitrary, the start end end times are still set at `Now() - 5 minutes` and `Now()`
+
+### Set up Datadog Cluster Agent to use DatadogMetric
+
+To activate usage of `DatadogMetric` CRD, follow these extra steps:
+
+1. Install the `DatadogMetric` CRD in your cluster.
+
+    ```shell
+    kubectl apply -f "https://raw.githubusercontent.com/DataDog/datadog-operator/master/deploy/crds/datadoghq.com_datadogmetrics_crd.yaml"
+    ```
+
+2. Update Datadog Cluster Agent RBAC manifest, it has been updated to allow usage of `DatadogMetric` CRD.
+
+    ```shell
+    kubectl apply -f "https://raw.githubusercontent.com/DataDog/datadog-agent/master/Dockerfiles/manifests/cluster-agent/agent-rbac.yaml"
+    ```
+
+3. Set the `DD_EXTERNAL_METRICS_PROVIDER_USE_DATADOGMETRIC_CRD` to `true` in the deployment of the Datadog Cluster Agent.
```

Comment:
would be nice to mention how to enable it using the helm chart and the operator since we have dedicated fields for this feature.

Reply:
The thing is, the whole document should be updated to mention Helm and Operator :(

Reply:
hah I see
 @cohenyair in case we want backlog this for the docs team maybe?

Reply:
I added this to the list for our planned operator docs updates